apps/opik-backend/pom.xml

```diff
@@ -141,6 +141,11 @@
             <artifactId>guicey-jdbi3</artifactId>
             <version>${dropwizard-guicey.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.github.dropwizard-jobs</groupId>
+            <artifactId>dropwizard-jobs-guice</artifactId>
+            <version>6.0.0</version>
```

Comment:
Make sure you use the latest version. It seems to be 6.0.2.

Reply:
Found this:
- https://mvnrepository.com/artifact/io.github.dropwizard-jobs/dropwizard-jobs-guice

Anyway, not a blocker, just try to be on the most recent one. Ignore my comment otherwise.

Reply:
Same, both point to 6.0.1